test/commands/modules/disable.spec.ts

```diff
@@ -19,7 +19,10 @@ jest.mock("../../../src/state/state-manager", () => {
     return {
         StateManager: function () {
             return {
-                excludeModule: excludeModuleMock
+                excludeModule: excludeModuleMock,
```

Comment:
Not seen `excludeModule` anywhere else is this relevant?

Reply:
Not sure I understand.